recommenders/models/sasrec/sampler.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import numpy as np
+from multiprocessing import Process, Queue
+
+
+def random_neq(left, right, s):
+    t = np.random.randint(left, right)
+    while t in s:
+        t = np.random.randint(left, right)
+    return t
+
+
+def sample_function(
+    user_train, usernum, itemnum, batch_size, maxlen, result_queue, SEED
```

Comment:
minor detail, typically we have all the inputs in lowercase, so SEED we would put it as seed